src/EditorFeatures/Test/Diagnostics/DiagnosticAnalyzerServiceTests.cs

```diff
@@ -496,9 +496,6 @@ public async Task TestHostAnalyzerErrorNotLeaking()
                         continue;
                     }
 
-                    var liveId = (LiveDiagnosticUpdateArgsId)e.Id;
-                    Assert.False(liveId.Analyzer is ProjectDiagnosticAnalyzer);
```

Comment:
removed IDs entirely.  it was just used for validating some things in tests at this point.   i want to remove the eventing system in IDiagService entirely.  but i'm doing it a step at a time.

Reply:
I believe these IDs were primary used for grouping/de-duping together same diagnostics on different TFMs of the same project so we get a single entry in the error list for them. This should be fine to remove now.